test_fixtures/require_test/file.gb

```diff
@@ -0,0 +1,6 @@
+require("file")
```

Comment:
`require_test` means that is for testing `require`
you should put this file in `text_fixtures` and name it `file_test.gb`